qiskit_nature/circuit/library/initial_states/hartree_fock.py

```diff
@@ -40,26 +40,65 @@ def __init__(
                            beta-spin electrons in the first and second number, respectively.
             qubit_converter: a QubitConverter instance.
         """
+        # Get the mapped/tapered hartree fock bitstring as we need it to match to whatever
+        # conversion was done by the given qubit converter
+        bitstr = hartree_fock_bitstring_mapped(
+            num_spin_orbitals, num_particles, qubit_converter, True
+        )
+        # Construct the circuit for this bitstring. Since this is defined as an initial state
+        # circuit its assumed that this is applied first to the qubits that are initialized to
+        # the zero state. Hence we just need to account for all True entries and set those.
+        qr = QuantumRegister(len(bitstr), "q")
+        super().__init__(qr, name="HF")
 
-        # get the bitstring encoding the Hartree Fock state
-        bitstr = hartree_fock_bitstring(num_spin_orbitals, num_particles)
+        for i, bit in enumerate(bitstr):
+            if bit:
+                self.x(i)
 
-        # encode the bitstring as a `FermionicOp`
-        label = ["+" if bit else "I" for bit in bitstr]
-        bitstr_op = FermionicOp("".join(label), display_format="dense")
 
-        # map the `FermionicOp` to a qubit operator
-        qubit_op: PauliSumOp = qubit_converter.convert_match(bitstr_op)
+def hartree_fock_bitstring_mapped(
+    num_spin_orbitals: int,
+    num_particles: Tuple[int, int],
+    qubit_converter: QubitConverter,
+    match_convert: bool = True,
+) -> List[bool]:
+    """Compute the bitstring representing the mapped Hartree-Fock state for the specified system.
 
-        # construct the circuit
-        qr = QuantumRegister(qubit_op.num_qubits, "q")
-        super().__init__(qr, name="HF")
+    Args:
+        num_spin_orbitals: The number of spin orbitals, has a min. value of 1.
+        num_particles: The number of particles as a tuple (alpha, beta) containing the number of
+            alpha- and  beta-spin electrons, respectively.
```

Comment:
```suggestion
            alpha- and beta-spin electrons, respectively.
```

Reply:
I don't think that has changed - if done here it won't be in main. As far as I can see in the output docs it makes no difference the spaces are sorted out between the words. So if we want to make this minor change I think its best done in main although as I said I don;t think it affects the output docs.